### PR TITLE
Postgresql aggregate functions

### LINKED ISSUE #4590: [Concept Entry] Postgresql aggregate-expressions
### Reason/inspiration (optional)

We would like a new entry on the `Aggregate Expressions` concept under PostgreSQL. The entry should go in a new file under `docs/content/postgresql/concepts/aggregate-expressions/aggregate-expressions.md`.

The entry should include:

- An introductory paragraph that describes the concept
- A �Syntax� section that includes the syntax for the concept (if applicable)
- An �Example� section that shows the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Johnnie-kay**
### Description
Created a new concept entry for Postgresql aggregate-functions.

### Issue Solved
Closes #4590 [Concept Entry] Postgresql aggregate-functions

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by Johnnie-kay** [comment]
Hello @Sriparno08, please review this PR. some chcks are still failling and i dont know how to correct it. Let us close this issue so i can get assigned to other issues.

**Comment by Sriparno08** [comment]
Hey @Johnnie-kay, your PR will be reviewed soon. In terms of the checks, the reviewers will handle them. So, don't worry about that.

File: content/postgresql/concepts/aggregate-functions/aggregate-functions.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+FROM sales
+GROUP BY Customer;
+```
+
+Breaking down the query:
+
+- `SELECT Customer` specifies the column name to retrieve from the table.
+- `SUM(Quantity * cost) AS Total_Spent` multiplies the `Quantity` and `cost` columns for each sale to calculate the total amount spent by the customer, and names the result column `Total_Spent`.
+- `FROM sales` specifies the table from which the query retrieves the data.
+- `GROUP BY Customer` clause groups the results by the `Customer` column.
```
Comment (Sriparno08) [inline content/postgresql/concepts/aggregate-functions/aggregate-functions.md:97]:
```suggestion
The `GROUP BY` clause groups the set of results, including the total amount spent by each customer, by the `Customer` column.
```

File: content/postgresql/concepts/aggregate-functions/aggregate-functions.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+This is the out put of the above query:
```
Comment (Sriparno08) [inline content/postgresql/concepts/aggregate-functions/aggregate-functions.md:99]:
```suggestion
Here is the output of the above query:
```

File: content/postgresql/concepts/aggregate-functions/aggregate-functions.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+```sql
+SELECT AVG(Quantity * Cost) AS Average_Spent FROM sales;
+```
+
+This query finds the average amount spent per sale in the sales table by calculating the product of the `Quantity` and `cost` columns for each sale and then averaging these values.
+
+## GROUP BY
+
+The `GROUP BY` clause in Postgresql is used to arrange identical data into groups. This clause is often combined with aggregate functions to perform operations on each group of data. Essentially, `GROUP BY` allows you to summarize data by grouping rows that have the same values in specified columns.
```
Comment (Sriparno08) [inline content/postgresql/concepts/aggregate-functions/aggregate-functions.md:82]:
```suggestion
### GROUP BY

The `GROUP BY` clause in PostgreSQL is used to arrange identical data into groups. This clause is often combined with aggregate functions to perform operations on each group of data. Essentially, `GROUP BY` summarizes data by grouping rows that have the same values in the specified columns.
```

**Review by Sriparno08 — CHANGES_REQUESTED** [review]
Hey @Johnnie-kay, thanks for contributing to Codecademy Docs!

I've listed all the modifications that you need to make in your entry below:

**Comment by Johnnie-kay** [comment]
Hello @Sriparno08, I had to deleted this branch to create another one... The name of the branch was creating a confusion for me. sorry for the inconviniences. I created another branch named 'aggregate-functions' for this issue.

**Comment by Johnnie-kay** [comment]
All the changes and corrections has been applied to the new branch as well. Please review it again.

**Comment by Johnnie-kay** [comment]
I think I found a wayb out of this confusion.

**Comment by Sriparno08** [comment]
Okay 👍

**Pull request closed without merging**